styles/Global.css

```diff
@@ -0,0 +1,113 @@
+html {
+	height: 100%;
+}
+h2 {
+	color: #f3f3f3;
+	font-family: raleway;
+	padding-left: 20px;
+	font-size: 36px;
+	font-weight: 700;
+	font-style: normal;
+	line-height: 43.2px;
```

Comment:
Line hight is normally expressed as multiples of font-size. Like `line-height: 2.1;`